varlink/mock.py

```diff
@@ -258,7 +258,7 @@ def __init__(
         else:
             self.name = name
         self.identifier = str(uuid.uuid4())
-        self.interface_description = None
+        self.interface_description: list[str] = []
```

Comment:
This change might break people assuming that an uninitalised `interface_description` is `None` and after `interface_description` is initalised it is a list with at least one element. This seems a bit silly, since an empty list is a perfectly fine boolean to encode this, so it doesn't need to be `None`, but I wonder if there is any usage of that out in the wild. The only thing I find for `interface_description is None` on GitHub is not varlink, so changing this is most likely safe.